dev/main.go

```diff
@@ -73,30 +74,58 @@ func (dev *DaggerDev) CLI() *CLI {
 	return &CLI{Dagger: dev}
 }
 
+// Return the Dagger source code, after applying codegen
+func (dev *DaggerDev) Source() *Directory {
```

Comment:
Hm, I'm not sure about switching *everywhere* to use this - that feels like we're adding extra cost to places that don't actually need it.

Could we instead rename this method to `GeneratedSource`/`SourceGenerated`, and then keep `Source` as before?